opcua/server/address_space.py

```diff
@@ -515,7 +515,7 @@ def set_attribute_value(self, nodeid, attr, value):
 
         for k, v in cbs:
             try:
-                v(k, value)
+                v(k, value, old)
```

Comment:
Yes, I am not a great fan of this too. Especially since it breaks the first callback when we create the subscription. I modified it so that the two arguments are the same, and therefore on a deadband filter we got no callback. I will see what I can make different. Thanks